conversion/tts_tokenizer_profiles.py

```
@@ -0,0 +1,170 @@
+    if len(matches) != 1:
+        raise ValueError(f"unsupported Magpie tokenizer layout: {list(order)}")
+    profile = matches[0]
+    if str(cfg.get("nemo_version", "")) != TOKENIZER_PROFILE_NEMO_VERSIONS[profile]:
+        raise ValueError(f"unsupported {profile} nemo_version: {cfg.get('nemo_version')!r}")
+    for name, target in TOKENIZER_TARGETS[profile].items():
+        tokenizer = tokenizers.get(name)
+        actual_target = tokenizer.get("_target_") if isinstance(tokenizer, dict) else None
+        if actual_target != target:
+            raise ValueError(
+                f"unsupported {profile} tokenizer target for {name}: " f"{actual_target!r}"
+            )
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)